src/core/include/openvino/op/add.hpp

```diff
@@ -43,6 +43,7 @@ class OPENVINO_API Add : public util::BinaryElementwiseArithmetic {
     OPENVINO_SUPPRESS_DEPRECATED_START
     bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Cab we remove old implementation?

Reply:
Same comment as below it looks like old implementation is required by tests.